### ISSUE TITLE
Unable to Access Network Log Folders

### ISSUE THREAD

**Issue by barricade823** [mreishman/Log-Hog#539]
Have some logs at a network path like the following, `\\SERVER\C:\FooBar\LOG`

If I'm trying to add the folder to the watchlist, the Add Folder dialog will see the folder and allow me to select it, and the WatchList entry for the folder shows no errors. When I try to save, the server calls to `savecheck.php?format=json` 19 times and then the loading dialog disappears. 

*From the server logs*
`[Tue Oct  2 11:44:28 2018] xxx.xxx.xx.xxx [200]: /core/php/saveCheck.php?format=json` x 19

If I try to navigate away from the watchlist page the Changes not Saved! dialog appears. 

If I click yes I want to leave without saving and navigate away from the Watchlist tab and then return, the Location changes to `\SERVER\C:\FooBar\LOG` and the entry now shows `u---------` and the File/Folder not found! icon.

I checked my network configuration to ensure that sharing was enabled between these two machines and I'm able to browse, see, and create server logs on the server from the machine running LogHog without any issue.

I'm digging a little bit more to see if I can find where the issue is coming from.

ps. hope you don't mind if I'm throwing some ideas around. If this deviates too much from what you're trying to do, just let me know. 😄

**Comment by mreishman** [comment]
Yeah, I created the function to trigger the error message but never called it anywhere...

I think it has something to do with the save logic breaking because of the \\\\.

Changing the local config file to use \\\\\\\\ I was able to get the log to load:

`'FileOrFolder23' => array('AlertEnabled' => 'true','AutoDeleteFiles' => '','ExcludeTrim' => 'false','FileInformation' => '{}','FileType' => 'folder','GrepFilter' => '','Group' => '','Location' => '/var/www/html/asdf\\\\sdfa/','Name' => '','Pattern' => '.log$','Recursive' => 'false','SaveGroup' => 'false')`

![image](https://user-images.githubusercontent.com/3468153/46373022-6b9efe80-c652-11e8-84ea-19885db161f3.png)


This might be fixed by checking for \\ and replacing it with \\\\ on the build of the save object?
